src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../theme/theme';
+
+.projectDetailsContainer {
+  width: 100%;
+  height: 100%;
+  overflow-y: auto;
+  overflow-x: hidden;
+  padding-right: 10px;
+  &::-webkit-scrollbar {
+    width: 10px;
+  }
+  &::-webkit-scrollbar-thumb {
+    border-radius: 100px;
+    background: rgba(224, 224, 224, 1);
+    height: 220px;
+  }
+  @include xsPhoneView {
+    &::-webkit-scrollbar {
+      display: none;
+    }
+    padding-right: 0px;
+    padding-bottom: 100px;
+  }
+}
+
+.projectInfoSection {
+  padding: 15px;
+  background: rgba(255, 255, 255, 1);
+  border-radius: 16px;
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  grid-area: 15px;
+  gap: 15px;
+  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.1);
+  @include xsPhoneView {
+    box-shadow: none;
+    margin-left: 0px;
+    padding: 15px 30px;
+  }
+}
```

Comment:
Move these styles to ProjectInfo.module.scss - that makes more sense in terms of heirarchy, as ProjectInfoSection lies within the /components folder which uses styles from /styles.